### ISSUE TITLE
Seren doesn't load any Movie or TV Show list

### ISSUE THREAD

**Issue by cs58292** [nixgates/plugin.video.seren#257]
I started having issues with Seren loading Movie or TV Show lists around 3 weeks ago.  I'm using a Fire TV version Fire OS 5.2.7.3 with Kodi 18.3.  My Real Debrid account is active and authorized.  The issue is I open Kodi, open Seren, go to Discover Movies, and that loads.  However anything after that doesn't.  I try Trending, and it just spins.  I've waited over 10 min and nothing loads.  Same thing with any other Movie list.  And I see the same issues for TV Shows.  I went to the add on tools and tried clearing all cache actions, no go.  I re-installed a4kscrapers, no go.  Re-authorized Real Debrid and Trakt, no go.  I cleared Kodi cache, no go.  I removed Kodi and re-installed with 18.7 and re-installed Seren.  Authorized Real Debrid and Trakt again.  Installed a4kScraper, but I'm still having the same issue.  I restarted my Router and Fire TV several times.  I also set my modem up as IP Passthough with a linksys router, still not working.  For DHCP I'm handing out googles DNS IPs of 8.8.8.8.  I use ATT Uverse with 100 mbps.  On my MacBook Air with Kodi 18.7 Seren is working so I don't suspect it's ISP or Router related.  I'm on the same Wi-Fi network going through the same router and ISP.   I went out and bought a new Nvidia Shield TV Pro, installed Kodi and then Seren and set it up, and I got it to work temporarily.  But now it's doing the same thing as my Fire TV.  The MacBook is still working though.   Attaching the logs from my Shield.  Around 18.57 is when I try loading the Trending Movie List, and 3 min later I give up and hit the back button.  I've waited up to 10 - 15 min before and nothing happens.  In this example it was around 3 min.  Any ideas why none of the Movie or TV lists are loading?[
[kodi_seren.log](https://github.com/nixgates/plugin.video.seren/files/4874230/kodi_seren.log)
](url).  Seems like something going on with Trakt syncing possibly?  I've authorized it.  I also tried to disable Trakt but still doesn't work.

**Comment by drinfernoo** [comment]
Rebuild the Trakt database from settings

**Comment by SerpentDrago** [comment]
do you have any old liked trakt lists that may have gone private ? could be a list you like is having some kinda issue causing trakt sync to fuck up on that device . the errors are pretty nasty in your log . 

I would suggest as a test , to start clean . do this exactly ~! 

1. disable seren , (do not uninstall . disable ! ) , restart kodi
2. NOW uninstall seren , choosing to wipe userdata when prompted 
3. Re install and resetup seren , this time use a new trakt account just to test 
4. see if you have any issues over the course of the next few days . 

Only thing i can thing of is some liked list your have is causing seren to mess up its db on those devices . in the future seren2.0 release db's have been overhauled and this should not be a issue  
.

You could also try this , Rebuild trakt database while you have debug enabled and post the log of you trying to rebuild the db . that can help pinpoint the list causing issues (if it is that )

**Comment by cs58292** [comment]
Thanks for the quick response and suggestions.   I tried rebuilding Trakt but that didn't work for me.  It also took around 2 hours to complete.  My trakt account is a couple years old but doesn't have a lot of content.  I noticed some other issues too, for example, downloads from google play would also take forever or not even start, both on my phone and Shield.   Other addons were having similar issues. 

To resolve my problem I had to disable IPV6 on my ATT BGW210 router.   I wasn't getting assigned the manual DNS IPs I set on my linksys EA6350 router due to some bug.  I tried downgrading the firmware with no luck.  So I went back to using the ATT router only and noticed I was being assigned an IPV6 address for DNS.    After disabling IPV6 google play is downloading apps quickly again and Movie and Show lists are loading as expected.  

Don't quite get why my MacBook was working this whole time but my other devices are working now too.
